packages/database/src/migrations/20210124073946-FixUNFPAPercentagesOfFacilitiesOfferingServicesReports-modifies-data.js

```diff
@@ -0,0 +1,79 @@
+'use strict';
+
+import { updateValues } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+async function getDashboardReportById(db, id) {
+  const { rows: dashboardReports } = await db.runSql(`
+      SELECT * FROM "dashboardReport"
+      WHERE id = '${id}';
+  `);
+  return dashboardReports[0] || null;
+}
+
+async function updateBuilderConfigByReportId(db, newConfig, reportId) {
+  return updateValues(db, 'dashboardReport', { dataBuilderConfig: newConfig }, { id: reportId });
+}
+
+const dashboardReportsConfig = {
+  ids: [
+    'UNFPA_Region_Percentage_Facilities_Offering_Services_ANC',
+    'UNFPA_Region_Percentage_Facilities_Offering_Services_Delivery',
+    'UNFPA_Region_Percentage_Facilities_Offering_Services_Family_Planning',
+    'UNFPA_Region_Percentage_Facilities_Offering_Services_PNC',
+  ],
+  oldDataBuilder: 'sumValuesPerQuarterByOrgUnit',
+  newDataBuilder: 'finalValuesPerQuarterByOrgUnit',
```

Comment:
```
'UNFPA_Region_Percentage_Facilities_Offering_Services_ANC',
'UNFPA_Region_Percentage_Facilities_Offering_Services_Delivery',
'UNFPA_Region_Percentage_Facilities_Offering_Services_Family_Planning',
'UNFPA_Region_Percentage_Facilities_Offering_Services_PNC',
```

These dashboards returns wrong data after converting to `tupaia` data. They should be taking final values per quarter for a facility instead of summing all the values during a quarter for a facility. (Original ticket for these dashboards: https://github.com/beyondessential/tupaia-config-server/issues/1166)

Reply:
Great catch Chris